### ISSUE TITLE
Release prerelease for xcode 10.2 / swift 5

### ISSUE THREAD

**Issue by koenpunt** [usabilla/usabilla-u4a-ios-swift-sdk#129]
To start testing our app with Xcode 10.2 / Swift 5 we require the Usabilla library to be compiled with Xcode 10.2 beta.

**Comment by torkvell** [comment]
Hi @koenpunt ,

Thanks for reaching out to support on this! Currently, we have a release for Xcode 10.1. We are working on a release for Xcode 10.2. I don't have an exact time estimate on the release date, but I'll make sure to notify you here once it has been released. 

In the meantime, let me know if you need anything else.

**Comment by GoranLilja** [comment]
Any updates on this, now that Xcode 10.2 and Swift 5 is officially released?

**Comment by jeroenleenarts** [comment]
Please please opensource the SDK for iOS, like the JS and Python libs to avoid this showstopper in the future. :)

**Comment by itoxa** [comment]
In the future such situations won't be a stopper since swift 5 is ABI-stable.

**Comment by leobarrospereira** [comment]
Also waiting for an update for Xcode 10.2 / Swift 5.

**Comment by GerritPost** [comment]
Me too, can't use usabilla now 😕

**Comment by celian-m** [comment]
Same

**Comment by Nonnus** [comment]
It really makes no sense at all that you dont care to update to you **commercial sdk** to be used by your **paying customers** in sync with well known ide updates. This is an ongoing issue that you caused by providing a pre compiled binary and by not releasing required updates as soon as betas are out and especially after a public release like yesterdays Xcode 10.2. 
It really feels that you are not taking the issue very seriously and by connection, you also dont seem to take your clients very seriously as well.

**ps:** From my part I will advise my company to drop your service / tool / sdk asap and in the meantime will reach our business contact to report about this sad reality you guys are causing with this very poor sdk support

**pps:** you have several ppl asking for some update to this topic for 12 hours already with **0 replies from your team**, I hope you can understand this is not acceptable at all for a commercial paid tool that is supposed to be correctly supported from your company

**Comment by michaelclearscore** [comment]
Very unprofessional. Commercial SDK. You knew Swift 5 was coming. Did not bother testing it on the betas. Don't have Swift 5 version ready on time. I cant compile the app. Wasted hours looking for workout. Ended up ripping Usabila from the app, just to get it to compile.

**Comment by keshcreations** [comment]
Hi all,

First of all, apologies for the delay in response from our side. The latest update I have for you regarding this is that our team is already working a compatible version of our SDK. I will keep a close eye on the progress they are making and update you all accordingly in this ticket.

Should you have any other questions, feel free to respond in this ticket or via support@usabilla.com.

**Comment by GerritPost** [comment]
I'm hoping you've worked all night... Any news?

**Comment by jeroenleenarts** [comment]
Let me guess.... Nope.

**Comment by keshcreations** [comment]
We have worked all night on this and the team is planning to release a new version of our SDK today. As soon as I have better news, I'll be back with an update.

**Comment by diederikh** [comment]
@keshcreations That's why Apple releases Xcode beta so you don't have to work all night ;-).

**Comment by keshcreations** [comment]
Hi all! I'm back again with an update. Our team just released a new SDK compatible with XCode 10.2:

https://github.com/usabilla/usabilla-u4a-ios-swift-sdk/releases

I do want to thank you for your patience as we were working on this release. I don't have to tell you all what to do now 😄but if we can help you with anything else, please let us know.

**Comment by gabrielmoraees** [comment]
@keshcreations Did you guys drop the support for iOS 9? Not working for me

**Comment by GerritPost** [comment]
Thanks, it's working now. @gabrielmoraees i'm using iOS 10+

**Comment by gabrielmoraees** [comment]
@GerritPost it works if I set the Podfile platform to 9.4. I don't know if there's any reason for this jump

**Comment by mthormann-dhi** [comment]
@keshcreations Will the pod version on Cocoapods be updated soon as well?

**Comment by keshcreations** [comment]
@gabrielmoraees afaik the new release should still be compatible with the iOS versions of the previous release. Let me double check to be sure.

**Comment by keshcreations** [comment]
@gabrielmoraees I can confirm that iOS 9 is still supported. However, I've been notified that it might take some time for the cocoapods server to aggregate the new build. Could you please try again?

**Comment by gabrielmoraees** [comment]
@keshcreations I saw you updated the Pod spec back to 9.0 here in this repository. Could you also update it in Cocoa Pods Repo? There's still 9.4

**Comment by mbernson** [comment]
I'm trying out version 6.1.0 in our app right now, and am getting a runtime warning.

```
file:///Users/teamsdk/.jenkins/workspace/sdk_swift-sdk_master-UDM67NCBBOBYKLFJF34FF5NVHPULDTM3QQPCZXIURE2F3I77F26Q/Usabilla/EventEngine/Leaves/Event.swift: runtime: Swift Language Runtime: implicit Objective-C entrypoint -[Usabilla.Event initWithName:] is deprecated and will be removed in Swift 4
```

@ Usabilla team: Note that you don't have to convert your source code to Swift 5 to get the ABI stability, a recompile should be enough given that you're on Swift 4.2 already. [source](https://twitter.com/jckarter/status/1111020867679453184)

**Comment by DavidAdade** [comment]
I'm also running into the issue @gabrielmoraees mentioned. 
`Specs satisfying the `Usabilla (= 6.1.0)` dependency were found, but they required a higher minimum deployment target.`

The deployment target is set to 9.0. Does support start at 9.4 now?

**Comment by muxor81** [comment]
Do you have any news on this? The pod repository still seems to have 9.4 set.

**Comment by keshcreations** [comment]
@mbernson @DavidAdade @muxor81 as our team is working on this, you can use the pod with version 9.0 under XCode 10.2 by specifying this in your podfile:

`pod ‘Usabilla’, :git => ’https://github.com/usabilla/usabilla-u4a-ios-swift-sdk.git', :tag => ‘v6.1.0’`

**Comment by torkvell** [comment]
Fix introduced in v6.1.0: https://github.com/usabilla/usabilla-u4a-ios-swift-sdk/releases. Ticket closed
